skaffold.yaml

```diff
@@ -0,0 +1,115 @@
+---
+apiVersion: skaffold/v4beta7
+kind: Config
+metadata:
+  name: gcmx
+build:
+  artifacts:
+    - image: eu.gcr.io/gardener-project/gardener/gardener-custom-metrics
+      ko:
+        dependencies:
+          paths:
+            - cmd/**
+            - example/**
+            - pkg/**
+            - vendor/**
+            - VERSION
+        ldflags:
+          - '{{.LD_FLAGS}}'
+        main: ./cmd/gardener-custom-metrics
+profiles:
+  - name: debug
+    activation:
+      - command: debug
+    manifests:
+      rawYaml:
+        - example/custom-metrics-deployment.yaml
+#      kustomize:
+#        paths:
+#          - example/provider-local/seed-kind2/skaffold
+    deploy:
+      kubectl: {}
+
+# TODO: Andrey: P1: cleanup
```

Comment:
I am a bit against having such big `TODO`s in the `main` branch

Reply:
Deleted.